### PR TITLE
Javascript Capstone

### PR DESCRIPTION
**Pull request by gemmen29**
In this project we added the following:

1. Use JavaScript to make websites dynamic and build basic single-page apps.
2. Use ES6 syntax.
3. Use ES6 modules.
4. Use callbacks and promises.
5. Use Webpack.
6. Apply JavaScript best practices and language style guides in code.
7. Use AAA pattern for unit tests.
8. Write units tests for a JavaScript app.
9. Follow Gitflow.
10. Solve simple git conflicts.
11. Send and receive data from an API.
12. Use API documentation.
13. Understand and use JSON.
14. Make JavaScript code synchronous.
15. Perform a code review for a team member.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,45 @@
...
+- SASS
 - JS
 - Bootstrap 5
 
+### Live version
+
+- live: [Javascript-Capstone](https://iwillteixeira.github.io/javascript-capstone/)
+
+### Project Video
+- [Video](https://drive.google.com/file/d/1kI4FmRN8IYxeZeRoA2kIq3AKwvdvPBeB/view)
```
Comment (D4nielJ) [inline README.md]: - [ ] Let's include in the video a highlight of a piece of code that you guys are proud of.

[OPTIONAL] I recommend including a small presentation of the students participating in the project, aswell as a small presentation of the main goal of the project.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,45 @@
...
--SASS
+- NODE
+- NPM
+- SASS
 
 ### Setup
 
 Dowload NPM and NODE
+
 ### Install
```
Comment (D4nielJ) [inline README.md:50]: [OPTIONAL] Please consider adding more detailed instructions of how to install and run the project: running dev scripts, tests, build, and deploy.

File: src/js/APIHelper.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  static baseURL = 'https://api.tvmaze.com/shows';
+
+  static async getAll(page = 1) {
+    try {
+      const shows = await fetch(`${APIHelper.baseURL}?page=${page}`).then(
+        (response) => response.json(),
+      );
+      return shows;
+    } catch {
+      console.log("Error: Couldn't fetch films data");
```
Comment (D4nielJ) [inline src/js/APIHelper.js]: [OPTIONAL] Good job catching errors when doing requests to API's. It is also a good practice to not just console.log these errors, instead displaying a message error on the application with feedback for the user leads to a better user experience.

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @gemmen29, @iwillteixeira, and @yigitm, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: :rocket: 
- Excellent project, one of the best JS capstones I've seen.
- You include correctly tests for the count functions
- I can add comments, likes, and make reservations as expected! :fireworks: :clap:
- Good use of ES6 syntax

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi Team🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**